test/integration/install_scripts_int_test.go

```diff
@@ -136,12 +124,28 @@ func scriptPath(t *testing.T, targetDir string) string {
 	return target
 }
 
-
 func expectStateToolInstallation(cp *termtest.ConsoleProcess) {
 	cp.Expect("Preparing Installer for State Tool Package Manager")
 	cp.Expect("Installation Complete", time.Second*40)
 }
 
+func expectStateToolWorks(cp *termtest.ConsoleProcess, activation bool) {
+	if activation {
+		cp.Expect("Creating a Virtual Environment")
+		cp.Expect("Quick Start", time.Second*60)
```

Comment:
Nit: These don't belong with "expect state tool works". Since you're only using this function in one place I'd just leave the code where it was.